ott/core/neuraldual.py

```diff
@@ -305,12 +306,11 @@ def step_fn(state_f, state_g, batch):
 
     return step_fn
 
-  def create_train_state(self, rng, model, optimizer, input):
-    """Create initial `TrainState`."""
-    params = model.init(rng, jnp.ones(input))["params"]
-    return train_state.TrainState.create(
-        apply_fn=model.apply, params=params, tx=optimizer
-    )
+  def to_dual_potentials(self) -> potentials.DualPotentials:
+    """Return the neural Kantorovich dual potentials."""
```

Comment:
Maybe say more explicitly

Create a :class:`ott.core.potentials.DualPotentials` object from two trained NN dual functions.

Reply:
Done as `Return the Kantorovich dual potentials from the trained functions.`, to fit 80 columns.